### ISSUE TITLE
Indication on deleted mail

### ISSUE THREAD

**Issue by Aivenho** [Thomas101/wmail#606]
WMail 2.1.0 on Ubuntu 16.04

Great piece of software. I just had one very annoying issue (with previous version as well, stayed after the update) - deleted unread mail was showing up as unread on tray and account icons. As it was deleted already from trash as well there were no way to change its status as read. 

Only solution was to remove account and add it back again...

**Comment by Thomas101** [comment]
Hi, if you completely re-launched WMail did the status change or was it fixed at that count?

**Comment by Aivenho** [comment]
With simple re-launch and with reboot nothing changed. Tray indicator was showing 1 and even. But when I click on it nothing happened, as mail was not available on system any more. Probably just cached somewhere and failed to delete himself after deletion.

**Comment by Thomas101** [comment]
Hmmm, this might be a bug on Googles side with the api. Are you using Gmail or Google Inbox? If you open the account in Gmail (if you're using inbox it has a link in the left hand menu) and search for `in:inbox in:unread ` in the search bar does it come back with no messages?

**Comment by Aivenho** [comment]
Yes, there are no such messages in Gmail any more. As I mentioned in initial post - glitch diapered when I removed particular account from Wmail and re-added it back.

**Comment by Thomas101** [comment]
Ok thanks for the info. I'll take a look into it and see if I can reproduce it my end. Thanks!
